src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <!-- These are wrapper project files for packaging.-->
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.0</TargetFrameworks>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <!-- Avoid trying to binplace non existent documentation file.-->
+    <EnableBinPlacing>false</EnableBinPlacing>
+  </PropertyGroup>
```

Comment:
Does this build a dll named "Microsoft.NETCore.Platforms.dll"?  Is there anything that can be done to prevent that?

Reply:
No it doesnt generate any dll. It generates only the deps file

Reply:
Interesting, where's the build magic that disables building anything?

Reply:
Its the ```<Project Sdk="Microsoft.Build.NoTargets">```

Reply:
I totally missed that.  I guess my eyes are used to ignoring the project element and looking for properties.  Thanks for educating me 😄